Gcpe.Hub.API.Tests/ControllerTests/TestData.cs

```diff
@@ -107,5 +107,18 @@ public static NewsRelease TestNewsRelease
             return message;
         }
 
+        public static SocialMediaPost CreateSocialMediaPost(string url, int sortOrder = 0)
+        {
+            var post = new SocialMediaPost
+            {
+                Id = Guid.Empty,

```

Comment:
Not needed

Reply:
I would rather be explicit so that anyone that looks at the test data will know the Id starts out empty.

Reply:
I prefer to have it there to be more explicit when people look at how the test post is created they will know that the id starts empty at a quick glance